kodi_voice/kodi.py

```diff
@@ -50,6 +50,14 @@ def sanitize_name(media_name, remove_between=False, normalize=True):
   name = name.strip()
   return name
 
+def sanitize_channel(name, lang='en'):
+  name = name.lower()
+  name = name.replace("+", " plus ")
+  name = re.sub(r"\sch\s", " channel ", name)
+  name = re.sub("^channel", "", name)
+  name = re.sub(r"(?<=\D)(?=\d)|(?<=\d)(?=\D)", " ", name)
+  name = words2digits(name, lang=lang)
```

Comment:
Fair enough.  In that case I'll change it to call sanitize_name from sanitize_channel and remove the words2digits call?  Are you happy with keeping the rest of sanitize_channel?

- switching + to plus (so channel 4+1 -> channel 4 plus 1)
- removing "channel" since it's so common it doesn't seem to be useful in the matching (i can combine the "ch" and "channel" regexps into one)
- putting breaks between numbers and words (4seven -> 4 seven)